tests/ui-components/Buttons/Button/Button.spec.tsx

```diff
@@ -1,7 +1,7 @@
 import theme from "@assets/styles/theme";
 import { ThemeProvider } from "@shopify/restyle";
 import { fireEvent, render } from "@testing-library/react-native";
-import { Button } from "@ui-components/Button/Button";
+import { Button } from "@ui-components/Buttons/Button/Button";
```

Comment:
co to za szalona ścieżka /Button/Button

Reply:
jest folder buttons, który ma folder button, a w tym folderze jest plik button.tsx, i tą końcówke można skipnąć, musi być tak jak jest xd

Reply:
no dobra